src/Libraries/CoreNodeModelsWpf/Converters/SelectionToTextConverter.cs

```diff
@@ -1,21 +1,30 @@
 ﻿using System;
 using System.Globalization;
 using System.Windows.Data;
+using DynamoConversions;
+using Dynamo.Wpf.Properties;
 
-namespace Dynamo.Wpf.Converters
+namespace Dynamo.Controls
 {
-    public class SelectionToTextConverter : IValueConverter
+    public class UnitToTextConverter : IValueConverter
```

Comment:
you may want to change the file name as well. I assume. SelectionToTextConverter is not used anywhere.